spec/System/OpensshGithubKeys/ConfigSpec.hs

```diff
@@ -10,75 +10,98 @@ import           Test.Hspec
 parserPrefs :: ParserPrefs
 parserPrefs = ParserPrefs "" False False True 80
 
-opts :: Options
-opts = Options "user" "stackbuilders" "team" ["user-to-add"] $ Just "/path/to/env"
-
 spec :: Spec
 spec = do
   describe "options" $ do
     it "uses the default organization" $ do
-      let defaults = [("organization", "stackbuilders")]
+      let defaults = [("organization", ["stackbuilders"])]
       let args = [ "--team=team"
                  , "--user=user-to-add"
                  , "--dotfile=/path/to/env"
                  , "user"
                  ]
-      getParseResult (execParserPure parserPrefs (options defaults) args) `shouldBe` Just opts
+      let expected = Options "user" "stackbuilders" "team" ["user-to-add"] "/path/to/env"
+      getParseResult (execParserPure parserPrefs (options defaults) args) `shouldBe` Just expected
 
     it "overrides the default organization" $ do
-      let defaults = [("organization", "other-organization")]
+      let defaults = [("organization", ["other-organization"])]
       let args = [ "--organization=stackbuilders"
                  , "--team=team"
                  , "--user=user-to-add"
                  , "--dotfile=/path/to/env"
                  , "user"
                  ]
-      getParseResult (execParserPure parserPrefs (options defaults) args) `shouldBe` Just opts
+      let expected = Options "user" "stackbuilders" "team" ["user-to-add"] "/path/to/env"
+      getParseResult (execParserPure parserPrefs (options defaults) args) `shouldBe` Just expected
 
     it "uses the default team" $ do
-      let defaults = [("team", "team")]
+      let defaults = [("team", ["team"])]
```

Comment:
Not quite sure I understand why this would be a list here. Is more than one team supported somewhere?